tests/sentry/utils/sdk_crashes/test_fixture.py

```diff
@@ -0,0 +1,269 @@
+from typing import Any, Mapping, Sequence
```

Comment:
fixtures belong in the `fixtures/` namespace.  the `tests/` namespace is for tests